robot-service/src/main/webapp/js/properties.js

```diff
@@ -0,0 +1 @@
+parser.url = "";
```

Comment:
It's because our concept. We want to filter our books by catagories. But now we cannot do this because we don't have api from db yet. It will be new feature in next days :facepunch: